### DISCUSSION TITLE
Threshold surpassed: 10000

### DISCUSSION THREAD

**Discussion by TheShubhendra** [lavalink-devs/Lavalink#586] [Q&A] [answered]
I'm getting this error very frequently after my bot crossed 200 servers, Lavalink also sometimes starts using 3GB+ RAM, I'm using digital Ocean droplet of 8GB RAM and it also tells about system overload, I think this is something related to load balancing, should I create a multiple nodes of Lavalink server? Is yes then how?

**Comment by JellyBrick** [comment, marked as answer]
Check this
https://github.com/freyacodes/Lavalink/issues/520#issuecomment-922927581
